src/main/java/taxi/controller/authentication/LogoutController.java

```diff
@@ -0,0 +1,16 @@
+package taxi.controller.authentication;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class LogoutController extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        request.getSession().invalidate();
+        response.sendRedirect(request.getContextPath() + "/login");
```

Comment:
```suggestion
        req.getRequestDispatcher(req.getContextPath() + "/login").forward(req, resp); 
```

Reply:
@vika-trach using `sendRedirect` is or here